assets/sass/backoffice/_icons.scss

```diff
@@ -0,0 +1,159 @@
+%icon-dashboard {
+    background: url("data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjQiIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggZD0iTTQgNEgxMFYxMkg0VjRaIiBzdHJva2U9IiMyNTJENDAiIHN0cm9rZS13aWR0aD0iMiIgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIiBzdHJva2UtbGluZWpvaW49InJvdW5kIi8+CjxwYXRoIGQ9Ik00IDE2SDEwVjIwSDRWMTZaIiBzdHJva2U9IiMyNTJENDAiIHN0cm9rZS13aWR0aD0iMiIgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIiBzdHJva2UtbGluZWpvaW49InJvdW5kIi8+CjxwYXRoIGQ9Ik0xNCAxMkgyMFYyMEgxNFYxMloiIHN0cm9rZT0iIzI1MkQ0MCIgc3Ryb2tlLXdpZHRoPSIyIiBzdHJva2UtbGluZWNhcD0icm91bmQiIHN0cm9rZS1saW5lam9pbj0icm91bmQiLz4KPHBhdGggZD0iTTE0IDRIMjBWOEgxNFY0WiIgc3Ryb2tlPSIjMjUyRDQwIiBzdHJva2Utd2lkdGg9IjIiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIgc3Ryb2tlLWxpbmVqb2luPSJyb3VuZCIvPgo8L3N2Zz4K") no-repeat center center;
+}
+
+%icon-my-country-profiles {
```

Comment:
This should be removed.

Reply:
done https://github.com/netgen/media-site/commit/f5531f7d7dea7ccb70e555fa113a0e299641e8fb